src/services/resource.js

```diff
@@ -770,6 +770,11 @@ module.exports = class resourceHelper {
 
 				if (item.meta?.notes) returnValue.notes = item.meta.notes
 
+				if (item.type === common.RESOURCE_TYPE_PROGRAM) {
```

Comment:
@priyanka-TL please add comments on why this if condition is required

Reply:
Only for program start date and end date required

Reply:
Added comment